pages/requester/my-tasks.tsx

```diff
@@ -110,7 +110,7 @@ export default function MyTasks() {
             size="small"
```

Comment:
To improve some of the wording, on this page could the secondary button be "Close" and primary button be "Responses"